slack_sdk/web/client.py

```diff
@@ -1555,34 +1555,53 @@ def emoji_list(self, **kwargs) -> SlackResponse:
         return self.api_call("emoji.list", http_verb="GET", params=kwargs)
 
     def files_comments_delete(
-        self, *, file: str, id: str, **kwargs  # skipcq: PYL-W0622
+        self, *, file: str, id: str  # skipcq: PYL-W0622
     ) -> SlackResponse:
         """Deletes an existing comment on a file.
 
         Args:
             file (str): The file id. e.g. 'F1234467890'
             id (str): The file comment id. e.g. 'Fc1234567890'
         """
-        kwargs.update({"file": file, "id": id})
-        return self.api_call("files.comments.delete", json=kwargs)
+        return self.api_call("files.comments.delete", json={"file": file, "id": id})
 
-    def files_delete(self, *, file: str, **kwargs) -> SlackResponse:
+    def files_delete(self, *, file: str) -> SlackResponse:
         """Deletes a file.
 
         Args:
             file (str): The file id. e.g. 'F1234467890'
         """
-        kwargs.update({"file": file})
-        return self.api_call("files.delete", json=kwargs)
+        return self.api_call("files.delete", json={"file": file})
 
-    def files_info(self, *, file: str, **kwargs) -> SlackResponse:
+    def files_info(
```

Comment:
<img width="896" alt="image" src="https://user-images.githubusercontent.com/8136030/128449248-9e77a696-9c0f-407a-8855-bd556711fb24.png">

I was using the above resource to define the optional arguments